services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluPaikkaServiceImpl.kt

```diff
@@ -11,6 +12,10 @@ class TormaystarkasteluPaikkaServiceImpl(val tormaystarkasteluDao: Tormaystarkas
      */
     override fun calculateTormaystarkasteluLuokitteluTulos(hanke: Hanke, rajaArvot: LuokitteluRajaArvot): List<Luokittelutulos> {
 
+        //if no geometries let's get out of here, this is invalid state
+        if (hanke.geometriat == null ||  hanke.geometriat?.id == null )
+            throw InvalidStateException("Hanke.geometriat should be set for hankeid ${hanke.id}")
```

Comment:
This would be better if it was an IllegalArgumentException - but no biggie.

Reply:
Changed